python/lib/dependabot/python/update_checker/pipenv_version_resolver.rb

```diff
@@ -78,6 +79,12 @@ def fetch_latest_resolvable_version_string(requirement:)
             SharedHelpers.in_a_temporary_directory do
               SharedHelpers.with_git_configured(credentials: credentials) do
                 write_temporary_dependency_files(updated_req: requirement)
+                # Initialize a git repo to appease pip-tools
```

Comment:
If we take action to handle git in my other comment, than that action needs to change here too...